engine/runtime/src/main/scala/org/enso/interpreter/runtime/util/DiagnosticFormatter.scala

```diff
@@ -155,7 +154,37 @@ class DiagnosticFormatter(
     if (isOutputRedirected) {
       return false
     }
-    return EnsoContext.get(null).isColorTerminalOutput;
+    return isColorTerminalOutput
+  }
+
+  /** Returns true if the output is a terminal that supports ANSI colors.
+    * See [[https://github.com/termstandard/colors/]] and [[https://no-color.org/]].
+    *
+    * Checking also system props, so that we can conveniently test this.
+    */
+  private def isColorTerminalOutput: Boolean = {
+    if (
```

Comment:
Having a `RuntimeOptions` option to influence `EnsoContext.isColorTerminalOutput` would seem less hackish to me.

Reply:
FYI: https://github.com/enso-org/enso/commit/a6ccc00284e940d0371af2dc41d4dcf9433b4dd9#r155527825

Reply:
Good point. Fixed in 1ab431c051ab415897daa3f0311409222ba002a6